instabot/bot/bot_filter.py

```diff
@@ -95,10 +95,17 @@ def search_stop_words_in_user(self, user_info):
 
 
 def search_blacklist_hashtags_in_media(self, media_id):
-    text = self.get_media_info(media_id)[0]['caption']['text']
+    media_info = self.get_media_info(media_id)
+    if media_info[0]['caption'] is None:
```

Comment:
Why do not you use a single-line expression? `text = media['caption']['text'] if media['caption'] else ''` ?

Reply:
thanks, done.